docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -0,0 +1,141 @@
+---
+approvers:
+- msau42
+- jsafrane
+title: Persistent Volume Claim Protection
+---
+
+{% capture overview %}
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+
+As of Kubernetes 1.9, persistent volume claims (PVCs) that are actively used by a pod can be protected from pre-mature deletion.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+- A v1.9 or higher Kubernetes must be installed.
+- As PVC Protection is a Kubernetes v1.9 alpha feature it must be turned on:
+1. [Admission controller](/docs/admin/admission-controllers/) must be started with the [PVC Protection plugin](/docs/admin/admission-controllers/#persistent-volume-claim-protection-alpha).
+2. All Kubernetes components must be started with the `PVCProtection` alpha features switched on.
+
+
+{% endcapture %}
+
+{% capture steps %}
+
+## PVC Protection Verification
+
+The below is done in GCE using a Storage Class, however, similar steps can be performed for any type of storage.
+
+Create a `StorageClass` for convenient storage provisioning:
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: slow
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-standard
+```
+
+There are two scenarios: a PVC that is being deleted is either used or not used by a pod.
+
+Firstly, the PVC is not used by a pod:
+- Create a PVC:
```

Comment:
Do you want to number each step?

Reply:
Well, I tried it originally and I tried it once again, but the rendering doesn't work for me. Either the numbering is screwed up (1., 1., 1. instead of 1., 2., 3.) or the verbatim (` ``` `) output is screwed up (not being verbatim at all). That's why I use the bullet points instead of numbers.

Reply:
You have to indent the yamls so that the numbering doesn't reset.  Take a look at [this](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/local-storage-overview.md) for an example.